src/com/rcv/ElectionConfig.java

```diff
@@ -47,10 +129,10 @@ public boolean validate() {
     } else if (this.tiebreakMode() == Tabulator.TieBreakMode.MODE_UNKNOWN) {
       valid = false;
     } else if (
-      overvoteLabel() != null &&
-      overvoteRule() != Tabulator.OvervoteRule.EXHAUST_IMMEDIATELY &&
-      overvoteRule() != Tabulator.OvervoteRule.ALWAYS_SKIP_TO_NEXT_RANK
-    ) {
+        overvoteLabel() != null &&
+            overvoteRule() != Tabulator.OvervoteRule.EXHAUST_IMMEDIATELY &&
+            overvoteRule() != Tabulator.OvervoteRule.ALWAYS_SKIP_TO_NEXT_RANK
+        ) {
```

Comment:
remove extra indent

Reply:
Done with top line. Remember that this is considered a continuation indent though, and thus should be indented 4 spaces per the Google standard.